rule/rule.go

```diff
@@ -547,7 +839,7 @@ func getArch(arch string) (string, uint32, error) {
 		}
 
 		switch runtimeArch {
-		case "aarch64", "x86_64", "ppc":
+		case "aarch64", "x86_64", "ppc64":
```

Comment:
Here I believe ppc was used instead of ppc64 by mistake

Reply:
Correct, ppc is 32.